include/fc/fwd.hpp

```diff
@@ -12,6 +12,7 @@ class fwd {
   public:
     template<typename U> fwd( U&& u );
     template<typename U, typename V> fwd( U&& u, V&& v );
+    template<typename U, typename V, typename X> fwd( U&& u, V&& v, X&& );
```

Comment:
is there any idea behind adding variable name for `U` and `V` but not for `X` ?

Reply:
No, just oversite.

Reply:
No. Added.